astexport/export.py

```diff
@@ -74,10 +78,15 @@ def visit_field_NameConstant_value(self, val):
 
     def visit_field_Num_n(self, val):
         if isinstance(val, int):
-            return {
-                self.ast_type_field: "int",
-                "n": val
-            }
+            data = {self.ast_type_field: "int", "n": val}
+
+            # JavaScript integers are limited to 2**53 - 1 bits,
+            # so we add a string representation of the integer
+            # if it's outside of the safe range
+            if val > JAVASCRIPT_MAX_SAFE_INT or val < JAVASCRIPT_MIN_SAFE_INT:
+                data["n_str"] = str(val)
```

Comment:
Why not always provide this so that people can count on it being there? (For example, one could pass it into `BigInt` to get a safe representation of it in JS.)